appcues/src/main/java/com/appcues/trait/AppcuesTraitException.kt

```diff
@@ -0,0 +1,3 @@
+package com.appcues.trait
+
+class AppcuesTraitException(message: String) : Exception("AppcuesTraitException: $message")
```

Comment:
wondering if it should be the handler's job to append any prefix onto the message?

Reply:
yeah thinking back I will just clear it out.